apps/chat/src/components/Files/FileManagerModal.tsx

```diff
@@ -280,7 +286,7 @@ export const FileManagerModal = ({
       <div className="flex flex-col gap-2 overflow-auto p-6">
         <div className="flex justify-between">
           <h2 id={headingId} className="text-base font-semibold">
-            {isInConversation ? t('Attach files') : t('Manage attachments')}
+            {isInConversation ? t('Attach files') : headerLabel}
```

Comment:
it looks like we don't need `isInConversation` if we have `customHeaderLabel`.
Instead of `isInConversation` you can set `customHeaderLabel={t('Attach files')}`
Could you remove `isInConversation`, please
```suggestion
            {customHeaderLabel ? customHeaderLabel : t('Manage attachments')}
```

Reply:
it looks like you need also `customButtonLabel` for button with `handleAttachFiles`